nitransforms/io/afni.py

```diff
@@ -36,33 +38,17 @@ def to_string(self, banner=True):
     @classmethod
     def from_ras(cls, ras, moving=None, reference=None):
         """Create an AFNI affine from a nitransform's RAS+ matrix."""
```

Comment:
Something like the following would help me understand the adjustment to the RAS:

```suggestion
        """Create an AFNI affine from a nitransform's RAS+ matrix.

        AFNI implicitly de-obliques image affine matrices before applying transforms, so
        for consistency we update the transform to account for the obliquity of the images.
        
        .. testsetup:
            >>> import pytest
            >>> pytest.skip()

        >>> moving.affine == ras @ reference.affine

        We can decompose the affines into oblique and de-obliqued components:

        >>> moving.affine == m_obl @ m_deobl
        >>> reference.affine == r_obl @ r_deobl
        
        To generate an equivalent AFNI transform, we need an effective transform (``e_ras``):
        
        >>> m_obl @ m_deobl == ras @ r_obl @ r_deobl
        >>> m_deobl == inv(m_obl) @ ras @ r_obl @ r_deobl
        
        Hence,
        
        >>> m_deobl == e_ras @ r_deobl
        >>> e_ras == inv(m_obl) @ ras @ r_obl
        """
```

I wrote it in Python, but LaTeX would be better. I just didn't want to try to do the syntax.